SCClassLibrary/Common/Files/SoundFile.sc

```diff
@@ -440,7 +440,7 @@ SoundFile {
 					}, "/n_end", server.addr, nil, ev[\id][0]).oneShot;
 				};
 				if (playNow) {
-					packet = server.makeBundle(false, {ev.play})[0];
+					packet = server.makeBundle(false, {ev.value.play})[0];
```

Comment:
I'm sorry, I was wrong.

In my (old?) environment, when I used the SoundFile-cue method, I always received an empty Event object. And it was fixed by changing the object to reference.

But now, I rechecked the code, SoundFile-cue method returns playable object.

So I cancel my pull request.

ps.
Thanks for your codes.

`ev = reference(ev ? ());`

This is a more sophisticated one of my intended code.

And this code, I wrote it for the purpose of creating a reference to the Event object.
```
(
f =  { |ev|
	ev = ev !? `_ ? `();
	ev
}
)

f.(); // returns `((  )) 
f.( (a: 3));  // returns `(( 'a': 3 ))
```